app/src/main/java/fr/free/nrw/commons/upload/UploadModel.java

```diff
@@ -209,6 +209,7 @@ public void next() {
     void setCurrentTitleAndDescriptions(Title title, List<Description> descriptions) {
         setCurrentUploadTitle(title);
         setCurrentUploadDescriptions(descriptions);
+        store.putString("Title",title.toString());
```

Comment:
Java developers have the habit of adding a space after the comma, would you mind adding it?
More about the coding style: https://github.com/commons-app/apps-android-commons/wiki/Code-style
You can commit and push the same branch, it will automatically update your pull request. Thanks :-)